admin_guide/cluster_management.rst

```diff
@@ -485,25 +491,104 @@ To set non-sensitive libpq connection parameters to be used for all node connect
+
+Next to these machine specific files you need these cluster or CA wide files available:
+ * ``/path/to/ca.crt``: Certificate of the Certificate Authority
+ * ``/path/to/ca.crl``: Certificate Revocation List of the Certificate Authority
+
+.. note::
+
+   The Certificate Revocation List is likely to change over time. Work with your security department on a mechanism to get up to date versions of the revocation list on to all nodes in the cluster in a timely manner. A reload of every node in the cluster is required after the revocation list has been updated.
+
+Once all files are in place on the nodes the following settings need to be configured in the Postgres configuration file:
+
+.. code-block:: ini
+
+   # the following settings enable the postgres server to enable ssl and configure the 
+   # server to present the certificate to clients when connecting over tls/ssl
+   ssl = on
+   ssl_key_file = '/path/to/server.key'
+   ssl_cert_file = '/path/to/server.crt'
+
+   # this will tell citus to verify the certificate of the server it is connecting to 
+   citus.node_conninfo = 'sslmode=verify-full sslrootcert=/path/to/ca.crt sslcrl=/path/to/ca.crl'
+
+After changing either restart the database or reload the configuration to apply these changes. A restart is required if a Citus version below 9.2.4 is used.
+
+Depending on the policy of the Certificate Authority used you might need or want to change ``sslmode=verify-full`` in ``citus.node_conninfo`` to ``sslmode=verify-ca``. For the difference between the two settings please consult `the official postgres documentation <https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS>`_.
+
+Lastly, to prevent any user from connecting via an unencrypted connection changes need to be made to ``pg_hba.conf``. Many Postgres installations will have entries allowing ``host`` connections which both allow SSL/TLS connections as well as plain TCP connections. By changing all ``host`` entries with ``hostssl`` entries only encrypted connections will be allowed to authenticate to Postgres. For full documentation on these settings take a look at `the pg_hba.conf file <https://www.postgresql.org/docs/current/auth-pg-hba-conf.html>`_ documentation on the official Postgres documentation.
+
+.. note::
+
+   When a trusted Certificate Authority is not available one can create their own via a self-signed root certificate. This is non-trivial and the developer or operator should seek guidance from their security team when doing so.
+
+To verify the connections from the coordinator to the workers are encrypted you can run the following query. It will show the SSL/TLS version used to encrypt the connection that the coordinator uses to talk to the worker:
+
+.. code-block:: postgresql
+
+  SELECT run_command_on_workers($$
+    SELECT version FROM pg_stat_ssl WHERE pid = pg_backend_pid()
+  $$);
+  
+  ┌────────────────────────────┐
```

Comment:
Sphinx says: "Could not lex literal_block as "postgresql". Highlighting skipped."

I'd advise splitting these into two adjacent code blocks, like

```rst
.. code-block:: postgresql

   SELECT foo;

::

   output
```